plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -323,15 +325,22 @@ def sequence_files(
                 )
 
     @large_file_check
-    def process(self, *, fname, in_str=None, config=None, formatter=None):
+    def process(
+        self,
+        *,
+        fname: str,
+        in_str: Optional[str] = None,
+        config: Optional[FluffConfig] = None,
+        formatter: Optional[OutputStreamFormatter] = None,
+    ):
         """Compile a dbt model and return the compiled SQL.
 
         Args:
-            fname (:obj:`str`): Path to dbt model(s)
-            in_str (:obj:`str`, optional): This is ignored for dbt
-            config (:obj:`FluffConfig`, optional): A specific config to use for this
+            fname: Path to dbt model(s)
+            in_str: fname contents using configured encoding
```

Comment:
The only meaningful change is that I changed the description for `in_str`. However, flake8 started complaining that some parameters didn't have docs, and I ended up moving the parameter types to be type annotations before I could get flake8 to be happy. 🤷